src/content/docs/style-guide/writing-docs/writer-workflow/github-troubleshooting.mdx

```diff
@@ -9,224 +9,281 @@ redirects:
-          "column": null
-        }
-      },
-      "source": "remark-mdx",
-      "ruleId": "unexpected-character",
-      "fatal": true
-    }
-    [
-    ```
+  The error does tell me that the doc with an error is in `/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide.mdx`. Other than that, all I know is that there is a `=` character messing things up somewhere. Cool, but this is a NRQL doc, there are a dozens of `=` characters.
 
-    In this case, the relvant info is:
+  So, I run `yarn verify-mdx src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide.mdx`. This returns the following:
 
-    ```
+  ```
+  reading src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/aqm-implementation-guide.mdx
+  {
+    "message": "Unexpected character `=` (U+003D) before name, expected a character that can start a name, such as a letter, `$`, or `_`",
+    "name": "284:227",
+    "reason": "Unexpected character `=` (U+003D) before name, expected a character that can start a name, such as a letter, `$`, or `_`",
     "line": 284,
     "column": 227,
-    ```
-
-    and
-
-    ```
+    "location": {
+      "start": {
+        "line": 284,
+        "column": 227,
+        "offset": 19129,
+        "index": 1
+      },
+      "end": {
+        "line": null,
+        "column": null
+      }
+    },
     "source": "remark-mdx",
     "ruleId": "unexpected-character",
     "fatal": true
-    ```
+  }
+  [
+  ```
 
-    This tells me that there is an unexpected character in line 284, column 227. Then I can go to that section of the doc and further troubleshoot, knowing exactly where the error comes from. In this case, I likely just need to wrap a NRQL example in quotes.
+  In this case, the relevant info is:
+
+  ```
+  "line": 284,
+  "column": 227,
+  ```
+
+  and
+
+  ```
+  "source": "remark-mdx",
+  "ruleId": "unexpected-character",
+  "fatal": true
+  ```
+
+  This tells me that there is an unexpected character in line 284, column 227. Then I can go to that section of the doc and further troubleshoot, knowing exactly where the error comes from. In this case, I likely just need to wrap a NRQL example in quotes.
   </Collapser>
 </CollapserGroup>
 
-## Issues with the local site [#local-site]
+</Collapser>
+<Collapser
+  className="freq-link"
+  id=""
+  title="Check for common syntax errors"
+>
+Check the changed files for some of these common markdown syntax errors:
+* Inconsistent indentation of the nav files
+* Incorrect front matter syntax
+  * If there's apostrophes and colons in frontmatter fields, surround them with quotes to avoid problems.
+* Missing closed brackets or tags
+* Poorly formatted image links
+  * Be careful when renaming images and their filename paths. A mismatch can cause the entire local build to fail.
+  * Be especially careful when dealing with image files that are imported.
```

Comment:
Something like this might be useful to add, just need to make sure that each image import has a blank line after

```suggestion
  * Be especially careful when dealing with image files that are imported.
  * Ensure a new line is between each image import statement
```